.travis.yml

```diff
@@ -178,13 +178,23 @@ jobs:
        apt-get -y install build-essential                                                 && \
        apt-get -y install git gfortran gcc g++ openmpi-bin libopenmpi-dev cmake           && \
        apt-get -y install libblas-dev liblapack-dev                                       && \
+       apt-get -y install libeigen3-dev                                                   && \
+       apt-get -y install python3-minimal python3-pip python3-numpy                       && \
+       pip3 install numpy                                                                 && \
+       apt-get -y install wget                                                            && \
+       wget https://sourceforge.net/projects/boost/files/boost/1.67.0/boost_1_67_0.tar.gz && \
```

Comment:
As you want python3 only, we _need_ to build from source : wget, tar, ...

Reply:
Why? Debian works without doing that

Reply:
Boost 1.67 exists in Debian testing:
https://tracker.debian.org/pkg/boost-defaults
why not running this part of the CI just on this distro ?

Reply:
Debian provides a package ` libboost-python-dev ` wich is built **for python2** , **not** for python **3**, that's why boost must be built-from-source for python3. If you can replace boost-build-from-source by an ` apt-get install `  would be good.

Reply:
Are you sure ? 
https://packages.debian.org/sid/libboost-python1.67-dev
% apt-cache show libboost-python1.67-dev|grep python3                                                                    Depends: libboost1.67-dev (= 1.67.0-17), libboost-python1.67.0 (= 1.67.0-17), python-dev, python3-dev

Reply:
AFAIR, does not work on my debian box for missing symbols : compil or tests fails. #248 should fail